.travis.yml

```diff
@@ -4,16 +4,14 @@ node_js:
   - "4"
   - "6"
   - "7"
+  - "8"
```

Comment:
At the end I found an unsolvable problem (due to the mantainer of a dependent module should do the fix) with node v10 :( so only node v8 is included. More detail in issue #55 

NTC (just informative)